doc/developers-guide/mongoose_wpool.md

```diff
@@ -0,0 +1,27 @@
+# mongoose_wpool
+
+All the [outgoing pools] configured by the `outgoing_pools` option are hidden behind `mongoose_wpool` API.
+Every pool is described by a tuple `{Type, Host, Tag, PoolOptions, ConnectionOptions}` (see [outgoing pools] for details about each element of the tuple).
+
+## Supervision tree
+
+![Supervision tree](mongoose_wpool_supervision.svg)
+
+* `mongoose_wpool_sup` supervisor for very type of the pool. Under it there can be many children of:
```

Comment:
`mongoose_wpool_sup` supervisor for every pool type. Under it there can be many children of: